apps/builder/app/builder/features/workspace/canvas-tools/outline/label.tsx

```diff
@@ -16,24 +16,22 @@ type LabelRefCallback = (element: HTMLElement | null) => void;
  * - else if instance height is more than 250px - bottom
  * - else inside-top - last resort because it covers a bit of the instance content
  */
-const useLabelPosition = (
-  instanceRect: Rect
-): [LabelRefCallback, LabelPosition] => {
+const useLabelPosition = (rect: Rect): [LabelRefCallback, LabelPosition] => {
   const [position, setPosition] = useState<LabelPosition>("top");
 
   const ref = useCallback(
     (element: null | HTMLElement) => {
-      if (element === null || instanceRect === undefined) {
+      if (element === null || rect === undefined) {
         return;
       }
       const labelRect = element.getBoundingClientRect();
       let nextPosition: LabelPosition = "top";
-      if (labelRect.height > instanceRect.top) {
-        nextPosition = instanceRect.height < 250 ? "bottom" : "inside";
+      if (labelRect.height > rect.top) {
```

Comment:
Why we check `height` here? And not the similar dimension like bottom or top?

Reply:
- didn't change this logic and actually forgot to revert the change I made in renaming
- tested logic works correctly, the change you are proposing breaks a use case